src/main/java/com/fvv/bookstore/util/DbPropertiesUtil.java

```diff
@@ -0,0 +1,70 @@
+package com.fvv.bookstore.util;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Load a DataBase properties file to connection factory.
```

Comment:
adjust this comment because this class can be used by other classes and not only by ConnectionFactory